mne/viz/tests/test_raw.py

```diff
@@ -284,75 +266,78 @@ def test_plot_raw_ssp_interaction(raw):
     raw.add_proj(projs)
     fig = raw.plot()
     # open SSP window
-    _fake_click(fig, fig.mne.ax_proj, [0.5, 0.5])
-    assert len(plt.get_fignums()) == 2
+    fig._fake_click((0.5, 0.5), ax=fig.mne.ax_proj)
+    assert browse_backend._get_n_figs() == 2
     ssp_fig = fig.mne.fig_proj
+    # ToDo: These gui-elements might differ in pyqtgraph.
     t = ssp_fig.mne.proj_checkboxes.labels
     ax = ssp_fig.mne.proj_checkboxes.ax
     assert _proj_status(ax) == [True, True, True]
     # this should have no effect (proj 0 is already applied)
     assert t[0].get_text().endswith('(already applied)')
     pos = np.array(t[0].get_position()) + 0.01
-    _fake_click(ssp_fig, ssp_fig.get_axes()[0], pos, xform='data')
+    fig._fake_click(pos, ssp_fig, ssp_fig.mne.ax_main, xform='data')
```

Comment:
only the main plot window has an `ax_main`.  It looks like in the SSP dialog figure we don't actually store a reference to the main axes, but a few lines above we've already tracked them down as `ax = ssp_fig.mne.proj_checkboxes.ax` 
```suggestion
    fig._fake_click(pos, ssp_fig, ax, xform='data')
```